doc/main.html

```diff
@@ -1235,6 +1249,272 @@ <h3>Git commit hooks</h3>
 
   </section>
 
+
+  <section class="annex" id="sec-document-status-workflow">
+    <h2>Document Status Workflow</h2>
```

Comment:
`pubDate` also needs to be updated when moving to `pub` state.

Reply:
I don't see where i missed this,